### ISSUE TITLE
devtools incompatible with R-devel?

### ISSUE THREAD

**Issue by bbolker** [r-lib/devtools#2226]
I have devtools 2.3.0 and R-devel (version 4.1.0, svn 78606).  I'm getting

>   devtools is incompatible with the current version of R. `load_all()` may function incorrectly.

Is this something I should worry about/expect trouble with, or is it a false positive?

I can track this as far as `pkgload:::extract_lang`, in these lines of code, but it's hard to debug because it happens somewhere in the package-loading process. Just staring at the code I have a hard time figuring out what it's doing ...

```
    if (is.call(x)) {
        res <- recurse(x)[[1]]
        if (top_level_call <- identical(sys.call()[[1]], as.symbol("extract_lang")) && 
            is.null(res)) {
            warning("devtools is incompatible with the current version of R. `load_all()` may function incorrectly.")
        }
        return(res)
    }
```

**Comment by jimhester** [comment]
This was fixed by a pkgload 1.1.0 was which released yesterday. (https://github.com/r-lib/pkgload/issues/124

**Comment by bbolker** [comment]
thanks!
